src/services/bootstrap/PreloadData.ts

```diff
@@ -52,21 +52,25 @@ export class PreloadData {
     this.defineEnvironment();
 
     await this.updateUser();
-    await this.updateUserProfile();
-    await this.updateBranding();
-    new ResourceFetcherService().prefetchResources(this.store.getState());
+    await this.updateJsonRpcMasterApi();
 
-    this.updateRestApiClient();
-    this.updateJsonRpcMasterApi();
+    const settings = await this.updateWorkspaceSettings();
```

Comment:
Not sure I got it why you get rid of `updateRestApiClient()` in favor of lazily initialize it, but haven't done the same for `updateJsonRpcMasterApi()`?

Reply:
I haven't use lazy initialization for ```updateJsonRpcMasterApi()``` because in the existing implementation it ignores a Promise.